lute/cli/commands/transform/init.luau

```diff
@@ -94,6 +94,11 @@ local function applyMigration(
 		migration.initialize(ctx)
 	end
 
+	local outputDirPath = if outputDir then pathLib.parse(outputDir) else nil
+	if outputDirPath and not fs.exists(outputDirPath) then
+		error("Output directory does not exist: " .. pathLib.format(outputDirPath))
+	end
```

Comment:
I think it's fine to make the output directory if it doesn't exist (and it's parent directories).